### DISCUSSION TITLE
error with python api httr

### DISCUSSION THREAD

**Discussion by rkb965** [posit-conf-2023/vetiver#34] [Q&A]
I am getting an error when trying to grab the prototype from:
https://colorado.posit.co/rsc/chicago-inspections-python

(the api link itself seems to have some sort of error?)

were other people able to get it to work?

**Comment by blairj09** [comment]
Yes that API seems to be down for the moment. However, you should be able to grab the prototype from the model that you publish to your workshop Posit Connect server.

**Comment by isabelizimm** [comment]
This API is up and running again! Thanks for the report.
